src/Form/Type/ChangePasswordType.php

```diff
@@ -18,7 +18,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('newPassword', RepeatedType::class, [
                 'type' => PasswordType::class,
                 'invalid_message' => 'The password fields must match',
-                'required' => true,
+                'required' => in_array('changepass', $options['validation_groups'], true),
```

Comment:
This form is specifically for changing passwords, so it does not make sense for the newPassword to ever not be required.